dbt_coverage/__init__.py

```diff
@@ -607,14 +674,18 @@ def fail_compare(coverage_report: CoverageReport, compare_path: Path):
 
 def do_compute(project_dir: Path = Path('.'), cov_report: Path = Path('coverage.json'),
                cov_type: CoverageType = CoverageType.DOC, cov_fail_under: float = None,
-               cov_fail_compare: Path = None):
+               cov_fail_compare: Path = None, model_path_filter: Optional[List[str]] = None):
     """
     Computes coverage for a dbt project.
 
     Use this method in your Python code to bypass typer.
     """
 
     catalog = load_files(project_dir)
+
+    if len(model_path_filter) >= 1:
```

Comment:
@followingell Is there a reason why `is not None` would not be appropriate here? Sorry if I missed anything obvious 🙂

Reply:
@mrshu Absolutely not, good question!

Originally I had used `is not None` as you say, however, this doesn't work due to a known issue with `typer`: https://github.com/tiangolo/typer/issues/410 whereby [an] _"Empty tuple [is] returned for Option instead of expected `None` when no values passed"_. As such, I have to use `len()` since until the issue is resolved `model_path_filter` will never be `None`.

Since you asked the question it's obvious that I could make the reason for my choice clearer, WDYT to the below suggestion?
```suggestion
    # TODO: Adjust len() to is not None after https://github.com/tiangolo/typer/issues/410 resolved
    if len(model_path_filter) >= 1:
```

Reply:
@mrshu FYI I removed the TODO comment _(see commit: ca47a9d5a837e168111b2405fa173c464f1fef20)_ since, IMO, instead this should be implemented as a project Issue post PR merge rather than within the code.